charts/tekton-apps/templates/_snippets.tpl

```diff
@@ -64,6 +64,17 @@ body.ref.startsWith('refs/heads/{{- $prefix -}}'),
 {{- end -}}
 {{- end }}
 
+{{/*
+Create the name for eventlistener
+*/}}
+{{- define "tekton-apps.eventlistenerName" -}}
+{{- if .suffix }}
+{{- printf "build-pipeline-event-listener-%s" .suffix }}
+{{- else }}
+{{- printf "build-pipeline-event-listener" }}
+{{- end }}
+{{- end }}
```

Comment:
make it simple pls

```go
{{- define "name" -}}
name: {{ . }}
{{ end -}}

{{- template "name" "foo" -}}
```

output:
```yaml
name: foo
```

Reply:
@dmitry-mightydevops do you mean like this?

```
{{- define "tekton-apps.eventlistenerName" -}}
name: {{ printf "build-pipeline-event-listener-%s" . }}
{{ end -}}

{{- template "tekton-apps.eventlistenerName" .Values.eventlistener.suffix -}}
```

Reply:
exactly